Mono.Nat/SocketGroup.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Mono.Nat
 {
-	class SocketGroup : IDisposable
+	public class SocketGroup : IDisposable
```

Comment:
I wouldn't want to expose this to Mono.Nat library users as it's an internal implementation detail, and it would be great to be able to change this around in future without breaking the API people rely on.

Reply:
that's ok - can work around this without an issue